specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotSecuritySolutionAnalytics.json

```diff
@@ -37,15 +37,15 @@
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/iotSecuritySolutions/{solutionName}/analyticsModels": {
       "get": {
         "x-ms-examples": {
-          "Get Security Solutions Analytics": {
+          "Get Security Solution Analytics": {
             "$ref": "./examples/IoTSecuritySolutionsAnalytics/GetIoTSecuritySolutionsSecurityAnalyticsList.json"
           }
         },
         "tags": [
-          "IoT Security Solutions Analytics"
+          "IoT Security Solution Analytics"
         ],
-        "description": "Security Analytics of a security solution",
-        "operationId": "IoTSecuritySolutionsAnalytics_GetAll",
+        "description": "Use this method to get IoT security Analytics metrics in an array.",
+        "operationId": "IotSecuritySolutionAnalytics_List",
```

Comment:
This will cause breaking changes for all SDKs. Could we generate a new API version?

Reply:
We did not published this SDK yet and there are no customers that uses this SDK yet. 
This is the new API version.